src/dbg/_exports.cpp

```diff
@@ -261,7 +263,16 @@ extern "C" DLL_EXPORT bool _dbg_addrinfoget(duint addr, SEGMENTREG segment, ADDR
                     {
                         if(instr.type == instr_branch)
                             continue;
-                        if(DbgGetStringAt(instr.arg[i].constant, string_text))
+
+						if (is_comparison_instr && 
+								instr.arg[i].constant >= 31 && instr.arg[i].constant < 255) // checks if character is printable
+						{
+							std::stringstream sstream;
+							sstream << std::hex << std::uppercase << instr.arg[i].constant;
+							sstream << ":" << "\"" << static_cast<char>(instr.arg[i].constant) << "\"";

```

Comment:
Limiting the printable subset (as per my comment above) to 126 would effectively make this a non-issue, unless I'm somehow mistaken.